main.go

```diff
@@ -97,6 +98,9 @@ func main() {
 	super.Add(servers.New())
 	addMgmtService(super)
 
+	// Restart previously running daemons
+	daemon.StartAllDaemons()
```

Comment:
Some of the rest handlers might have to communicate with these daemons and as soon as the rest server is up, glusterd2 will accept incoming requests. So it'll be safer to start the daemons first and then start supervisor chain of servers.

Reply:
The daemons need the sunrpc server to be up though. I think its acceptable that calls can fail for a little while.